templates/application.yaml

```diff
@@ -0,0 +1,35 @@
+{{- $dot := . }}
+{{- $wave_counter := 0 }}
+{{- range $cluster_name, $cluster_address := .Values.clusters }}
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: {{ include "appofapps.name" $dot }}-{{ $cluster_name }}
+  namespace: {{ $.Release.Namespace }}
+  annotations:
+    argocd.argoproj.io/sync-wave: {{ $wave_counter | quote }} 
+  labels: {{ nindent 3 (include "appofapps.labels" $dot) }}
+spec:
+  destination:
+    server: {{ $cluster_address | quote }}
+    namespace: default
```

Comment:
should it be configurable or `{{ $.Release.namespace }}`?

Reply:
no; it's the target namespace. i'll add a config item.